etc/inc/notices.inc

```diff
@@ -350,6 +350,10 @@ function send_smtp_message($message, $subject = "(no subject)", $force = false)
 
 	if ($config['notifications']['smtp']['fromaddress']) {
 		$from = $config['notifications']['smtp']['fromaddress'];
+		preg_match('/[\w\s]+/', $from, $matches);
+		$from_display = trim($matches[0]);
+		if(preg_match('/(?:<)(.+)(?:>)$/', $from, $matches))
+			$from = $matches[1];
```

Comment:
Yes, you're right. What do you think about add another field on the page to
fill $from_display?